Updated ZHA configuration and device adding (#9782)
home-assistant/home-assistant.github.io: source/_components/zha.markdown

  - [ConBee USB adapter from Dresden-Elektronik](https://www.dresden-elektronik.de/conbee/)
  - [RaspBee Raspberry Pi Shield from Dresden-Elektronik](https://www.dresden-elektronik.de/raspbee/)

## Configuration - GUI

> Visit `http://homeassistant-ip:8123/config/integrations/dashboard` which will put you on the `configuration` -> `integrations` page.

Reviewer: Please avoid this fake URL. Instead, describe it. For example: From the Home Assistant frontend to the the **Configuration** menu and click on **Integrations**. Or similar.
Please avoid the use of backticks here, since it is not describing any code like.

Use the plus button in the bottom right to add a new integration called `ZHA`.

In the popup:

>   - USB Device Path - on a rpi will be something like `/dev/ttyUSB0`

Reviewer: Avoid the use of "rpi".

>   - Radio type - select device type

Reviewer: Which device?

  - Submit

> The success dialog will appear or an error will be displayed in the popup. An error is likely if homeassistant can't access the usb device (see troubleshooting) or your device is not upto date.

Reviewer: ```suggestion
The success dialog will appear or an error will be displayed in the popup. An error is likely if Home Assistant can't access the USB device or your device is not up to date (see troubleshooting).
```

> The success dialog will appear or an error will be displayed in the popup.
That sentence is not really working... try to restructure/reword it.

In case you want to add Philips Hue bulbs that have previously been added to another bridge, have a look at: [https://github.com/vanviegen/hue-thief/](https://github.com/vanviegen/hue-thief/)

> ## Adding devices

Reviewer: ```suggestion
## Adding devices

```
missing newline

> Go to the `Configuration` page and select the `ZHA` element that was added by the configuration steps above

Reviewer: ```suggestion
Go to the **Configuration** page and select the **ZHA** integration that was added by the configuration steps above.
```

> Click `ADD DEVICES` to start a scan for devices.

Reviewer: ```suggestion
Click on **ADD DEVICES** to start a scan for new devices.
```

> Reset your zigbee devices according to your manufacturer instructions (lights turn on and off upto 10 times, switches using the reset pin)

Reviewer: ```suggestion
Reset your ZigBee devices according to the device instructions provided by the manufacturer (e.g.,  turn on/off lights up to 10 times, switches usually have a reset button/pin).
```

> Devices will now become visible and can be added to `Areas`

Reviewer: ```suggestion
```

Removing this sentence does not change the text in any way, so let us remove it.

```bash
sudo apt-get purge modemmanager
```

> ### Can't connect to usb device and using docker

Reviewer: ```suggestion
### Can't connect to USB device and using Docker

```

Missing newline + spelling

> If you can't connect as you are using docker you must forward your device from the host machine to the docker instance. This can be achieved by adding the device mapping to the end of the startup string or ideally using docker compose:

Reviewer: ```suggestion
If you are using Docker and can't connect, you most likely need to forward your device from the host machine to the Docker instance. This can be achieved by adding the device mapping to the end of the startup string or ideally using docker compose.
```

> #### Docker Compose

Reviewer: ```suggestion

#### Docker Compose

```
Missing newlines

> Install docker-compose for your platform (rpi - `sudo apt-get install docker-compose`).

Reviewer: ```suggestion
Install Docker Compose for your platform (rpi - `sudo apt-get install docker-compose`).
```
The Install command is not limited to the Raspberry Pi, also avoid abbreviations like "rpi".

> Create a docker-compose.yml with the following data:

Reviewer: ```suggestion
Create a `docker-compose.yml` with the following data:

```

This is a bit scary. A new user might think he needs to create a new (additional) Docker Compose setup?

Author: I agree it is a bit scary, its more like an optional step. I've nested it in the trouble shooting section as it might not apply to all users, only those that choose to use docker.
